experiments/integrated/ozark/ozark_met_drivers_FLUXNET.jl

```diff
@@ -99,8 +99,13 @@ atmos_u = Spline1D(seconds, WS[:])
 LW_IN_spline = Spline1D(seconds, LW_IN[:])
 SW_IN_spline = Spline1D(seconds, SW_IN[:])
 atmos_h = FT(32)
-precipitation_function(t::FT) where {FT} = p_spline(t) < 0.0 ? p_spline(t) : 0.0 # m/s
-snow_precip(t) = eltype(t)(0) # this is likely not correct
+
+function precipitation_function(t)
```

Comment:
Im not sure we need this. the eltype of the spline is Float64, so I dont think this gains us anything over using eltype of t?

Reply:
You're right, we can remove it since we wrap the value in FT after it's returned